sdk/src/wallet/core/operations/stronghold_backup/mod.rs

```diff
@@ -134,7 +134,9 @@ impl Wallet {
                 .map_err(|_| WalletError::Backup("invalid secret_manager"))?;
 
             // Copy Stronghold file so the seed is available in the new location
-            fs::copy(backup_path, new_snapshot_path)?;
+            if backup_path != new_snapshot_path {
```

Comment:
I don't think this will always work, since the equality just checks the components are the same, and they could be the same path but one is relative and one is absolute, for instance.

Reply:
Good catch